### PR TITLE
shim-v2/build.sh: Only build runtime-rs for the supported arches

### PR DESCRIPTION
**Pull request by fidencio**
Right now only x86_64 and ARM are supported.

Fixes: #6142

Signed-off-by: Fabiano Fidêncio <fabiano.fidencio@intel.com>

### CODE REVIEW

**Comment by fidencio** [comment]
Maybe a better approach would be to do exactly what we do for s390x in the runtime-rs Makefile.  Let me switch.

File: tools/packaging/static-build/shim-v2/build.sh [hunk 1]
```diff
@@ -27,15 +27,17 @@ if [ ${arch} = "ppc64le" ]; then
...
-sudo docker run --rm -i -v "${repo_root_dir}:${repo_root_dir}" \
-	-w "${repo_root_dir}/src/runtime-rs" \
-	"${container_image}" \
-	bash -c "git config --global --add safe.directory ${repo_root_dir} && make PREFIX=${PREFIX} QEMUCMD=qemu-system-${arch}"
-
-sudo docker run --rm -i -v "${repo_root_dir}:${repo_root_dir}" \
-	-w "${repo_root_dir}/src/runtime-rs" \
-	"${container_image}" \
-	bash -c "git config --global --add safe.directory ${repo_root_dir} && make PREFIX="${PREFIX}" DESTDIR="${DESTDIR}" install"
+if [ "${arch}" != "ppc64" ] || [ "${arch}" != "s390x" ]; then
```
Comment (liubin) [inline tools/packaging/static-build/shim-v2/build.sh:30]: Should

```diff
- if [ "${arch}" != "ppc64" ] || [ "${arch}" != "s390x" ]; then
+ if [ "${arch}" != "ppc64" ] && [ "${arch}" != "s390x" ]; then
```

**Review by studychao — APPROVED** [review]
Thanks:)

File: tools/packaging/static-build/shim-v2/build.sh (hunk 1 above)
Comment (studychao) [reply tools/packaging/static-build/shim-v2/build.sh:30]: +1

**Review by Tim-0731-Hzt — APPROVED** [review]

**Comment by BbolroC** [comment]
@fidencio has allowed me to take over this PR and finalize it. I got a target `install` included in the conditional branch as well because `make install` is called for `CCv0` (to which the PR is supposed to be back-ported).
Unless the target is included, a target `runtime` is triggered by a `install` - `install-runtime` chain, which breaks the CI on s390x and ppcle64.

**Review by cmaf — APPROVED** [review]

**Pull request merged**